client/jetpack-cloud/sections/partner-portal/issue-license-v2/review-licenses/pricing-summary.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import { useSelector } from 'calypso/state';
+import { getProductsList } from 'calypso/state/products-list/selectors';
+import { getTotalInvoiceValue } from '../lib/pricing';
+import type { SelectedLicenseProp } from '../types';
+
+export default function PricingSummary( {
+	selectedLicenses,
+}: {
+	selectedLicenses: SelectedLicenseProp[];
+} ) {
+	const translate = useTranslate();
+
+	const userProducts = useSelector( getProductsList );
+	const { discountedCost, actualCost } = getTotalInvoiceValue( userProducts, selectedLicenses );
+
+	const selectedLicenseCount = selectedLicenses
+		.map( ( license ) => license.quantity )
+		.reduce( ( a, b ) => a + b, 0 );
+
+	const handleCTAClick = useCallback( () => {
+		// TODO: Add tracking
+		// TODO: Handle onclick
+	}, [] );
+
+	const link = '#link'; // TODO: Add link
+
+	const currency = selectedLicenses[ 0 ].currency; // FIXME: Fix if multiple currencies are supported
+
+	return (
+		<>
+			<div className="review-licenses__pricing">
+				<span className="review-licenses__price">
```

Comment:
[non-blocker]I suggest to name this `review-licenses__pricing-discounted` and the other `review-licenses__pricing-original` to match the corresponding content.